src/installer.py

```diff
@@ -69,26 +76,23 @@ def dlstatus():
                 
                 pyotherside.send('whatState',"=> "+ str(size) +"/687180204 bytes (" + str(round(size / 687180204 * 100, 2)) + "%)")
                 
-                if size >= 680000000:
-                    pyotherside.send('whatState',"=> Almost done!")
+                if size == 687180200:
```

Comment:
This is not going to work, or rather will work only if by chance the downloaded bytes amount will hit exactly that number (not likely)

Reply:
Uhhh should i increase it? Kinda don't want to wait 10k seconds unconditionally